Firestore/core/src/firebase/firestore/nanopb/byte_string.h

```diff
@@ -103,7 +107,7 @@ class ByteString : public util::Comparable<ByteString> {
   }
 
   bool empty() const {
-    return size() == 0;
+    return bytes_ == nullptr || bytes_->size == 0;
```

Comment:
line 122:
If you change things such that get() always returns non-nullptr, then you could alter the signature from `const pb_bytes_array_t* get() const` to `const pb_bytes_array_t& get() const`